.github/workflows/smoke.yml

```diff
@@ -64,12 +64,12 @@ jobs:
     if: github.event.pull_request.draft == false
     strategy:
       matrix:
-        snap_version: ["latest/stable", "latest/beta"]
+        snap_version: ["2.8/stable"]
```

Comment:
Think we should replace this with an environment variable - don't need the matrix for a single version

Reply:
I think it's better to keep it as an array and we should probably consider to add `["2.8/stable", "2.9/stable"]` for develop branch.